File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.py
class Solution:
    def reverseDegree(self, s: str) -> int:
        n = len(s)
        sumi = 0
        for i in range(n):
            revidx = 26 - (ord(s[i]) - ord('a'))
            sumi += revidx * (i + 1)
        return sumi
        

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.cpp
class Solution {
public:
    int reverseDegree(string s) {
        int n  = s.size();
        int sum = 0;
        for(int i = 0 ; i < n ; i++){
            int revidx = 26 - (s[i] - 'a');
            sum += revidx * (i + 1);
        }
        return sum;
    }
};